tap_hubspot/client.py

```diff
@@ -124,12 +100,10 @@ def request(self,
                 path=None,
                 url=None,
                 **kwargs):
-        
-        """
+    
         if url is None:
             self.check_and_renew_access_token()   
```

Comment:
Hmm... this block looks redundant because the callers of `HubspotClient.get` never pass in a url.

Reply:
Agree! But left the logic in place, for future needs (if any). 

FYI - The tap for zoom has a similar block